File: Day25/Solution.java
public class Solution {

    public static void main(String[] args) {
        /* Enter your code here. */
        Scanner sc = new Scanner(System.in);
        int t = sc.nextInt();
        for (int i=0; i < t; i++){
            int flag = 0;
            int num = sc.nextInt();
            if (num < 2){
                System.out.println("Not prime");
            }
            else
            {
                for (int j=2; j <= Math.sqrt(num); j++){
                    if (num%j == 0){
                        flag =1;
                        break;
                    }
                }
                if (flag == 0){
                    System.out.println("Prime");
                }
                else {
                    System.out.println("Not prime");
                }
            }

        }

    }
}

File: Day25/Solution.py
import math

def check_prime(num):
    if num is 1:
        return "Not prime"
    sq = int(math.sqrt(num))
    for x in range(2, sq+1):
        if num % x is 0:
            return "Not prime"
    return "Prime"


t = int(input())
for i in range(t):
    number = int(input())
    print(check_prime(number))
